drivers/syslog/syslog_device.c

```diff
@@ -73,26 +73,10 @@ enum syslog_dev_state
   SYSLOG_OPENED,            /* SYSLOG device is open and ready to use */
 };
 
-/* This structure contains all SYSLOGing state information */
-
-struct syslog_dev_s
```

Comment:
Don't remove, let's change to:
```
* This structure contains all SYSLOGing state information */

struct syslog_dev_s
{
  struct syslog_channel_s channel;
  uint8_t      sl_state;    /* See enum syslog_dev_state */
  uint8_t      sl_oflags;   /* Saved open mode (for re-open) */
  uint16_t     sl_mode;     /* Saved open flags (for re-open) */
  sem_t        sl_sem;      /* Enforces mutually exclusive access */
  pid_t        sl_holder;   /* PID of the thread that holds the semaphore */
  struct file  sl_file;     /* The syslog file structure */
  FAR char    *sl_devpath;  /* Full path to the character device */
};
```

Reply:
I did it like this. `struct syslog_dev_state_s` is removed and `struct syslog_dev_s` is now private.